static/bidder-params/grid.json

```diff
@@ -7,7 +7,19 @@
     "uid": {
       "type": "integer",
       "description": "An ID which identifies this placement of the impression"
+    },
+    "keywords": {
+      "type": "object",
+      "description": "Keywords",
+      "properties": {
+        "site": {
+          "type": "object"
+        },
+        "user": {
+          "type": "object"
+        }
+      }
     }
   },
-  "required": []
+  "required": ["uid"]
```

Comment:
Do you have any statistics on how many requests come in currently without a `"uid"` parameter? There was a design choice that any request that does not pass validation on any one of the bidder JSON validation schemas fails entirely, not just for the one bidder. This was done as when a publisher sets up a new configuration with multiple bidders, a bidder parameter failure on one bidder may be masked by all the successful bids from other bidders. We would rather the requests were correct from the start.

This means any current bidder configs that do not include `uid` will fail to fetch ads from any bidder. This will break publisher configs across all bidders, not just yours. The preferred way to add a new required field is to initially list it as optional in the schema, and have the adapter return an error if it is missing. An adapter returning an error outside of JSON schema validation will not invalidate the whole request. Then once we all have some confidence that all the publishers have updated their configs, put in a followup PR to change the schema to required.